src/components/Header.js

```diff
@@ -30,7 +30,7 @@ const NavLink = styled(Link)`
 
 const Header = () => (
   <HeaderWrapper>
-    <Link to="/">
+    <Link to="/" id="header-logo-link">
```

Comment:
What is this id used for?

Reply:
left over from the way I was testing... removing it!